doc/dev/developing.md

```diff
@@ -46,6 +46,40 @@ In short, these are our general requirements:
 ## Detailed specifications
 
 Please have a look at the following documents for more detailed architecture specifications:
+
 - [Physical board](board.md)
 - [FPGA configuration](fpga.md)
 - [Software](software.md)
```

Comment:
We can remove these bullet points as they are remnants from the old documentation

Reply:
Thanks Colin!